documentation/optee_design.md

```diff
@@ -609,8 +609,9 @@ to as 'Trusted Applications'. For most cases, user mode TAs are preferred.
 
 ### Pseudo Trusted Applications
 
-These are added directly to the OP-TEE core tree in, eg, `core/arch/arm/pta`,
-and are built along with and statically built into the OP-TEE core blob.
+These are implemented directly to the OP-TEE core tree in, eg,
+`core/arch/arm/pta`, and are built along with and statically built into the
```

Comment:
(...) and are built along with and statically **~~built~~ linked** into the (...)

Reply:
This sentence isn't the best, but exchanging **built into** with **linked into** doesn't make it better. 
IMO, the words **linked into** implies some additional isolation compared to the other code.